inception-imls-opennlp/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/opennlp/doccat/OpenNlpDoccatRecommender.java

```diff
@@ -215,9 +211,11 @@ public EvaluationResult evaluate(List<CAS> aCasses, DataSplitter aDataSplitter)
             Type sentenceType = getType(cas, Sentence.class);
             Type tokenType = getType(cas, Token.class);
 
-            Map<AnnotationFS, List<AnnotationFS>> sentences = indexCovered(
+            Map<AnnotationFS, Collection<AnnotationFS>> sentences = indexCovered(
                     cas, sentenceType, tokenType);
-            for (Entry<AnnotationFS, List<AnnotationFS>> e : sentences.entrySet()) {
+            Collection<AnnotationFS> annotations=new ArrayList<>();//实例化的具体类型要看selectCovered(annotationType, sentence)的返回
```

Comment:
If I understand the change correctly, you try here to unroll two nested loops. Since this refactoring is rather complex, I would like to see a test case that verifies that the change is not changing the behavior of the code.